.travis.yml

```diff
@@ -26,6 +26,7 @@ before_script:
   ## we use the '&' ampersand which tells
   ## travis to run this process in the background
   ## else it would block execution and hang travis
+  - npm i -g npm@latest
```

Comment:
I believe this has to go in a `before_install` since Travis automatically runs `install` before any scripts. https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Using-a-specific-npm-version